docs/assets/js/docs.js

```diff
@@ -1,16 +1,22 @@
-m_triggers = document.querySelectorAll("[data-trigger-modal]");
+/* Modals */
+const modalTriggers = document.querySelectorAll("[data-trigger-modal]");
 
-for(var i=0; i < m_triggers.length; i++) {
-  mt = m_triggers[i];
+for (const i = 0; i < modalTriggers.length; i++) {
+  modalTrigger = modalTriggers[i];
 
-  mt.addEventListener("click", function(e) {
+  modalTrigger.addEventListener("click", function (e) {
     modal = document.querySelector(this.getAttribute("data-trigger-modal"));
 
-    if(!modal.classList.contains("is-with-backdrop"))
+    // Set appropriate on-screen position for modals,
+    // which do not have .is-with-backdrop, and are
+    // therefore position absolute not fixed.
+    if(!modal.classList.contains("is-with-backdrop")) {
       modal.style.top = document.scrollingElement.scrollTop + "px";
```

Comment:
No changes necessary here, but we'll want to consider for the JS standards whether we want to allow implicit type coercion like this or whether we require an explicit `toString()` or template literal use.

Reply:
Gotcha.